### ISSUE TITLE
Can't import configuration from file in v0.32.5-alpha

### ISSUE THREAD

**Issue by spacecowboy** [airbytehq/airbyte#8231]
## Enviroment
- **Airbyte version**:  v0.32.5-alpha
- **OS Version / Instance**: GKE
- **Deployment**:  Kubernetes
- **Severity**: Medium
- **Step where error happened**: Import configuration

## Current Behavior
I drag an exported config over to the import window, and when I click and select the file, nothing then happens. The submit button never becomes clickable.

## Expected Behavior
I should be able to import a configuration

## Logs
Nothing out of the ordinary is printed in the log

## Steps to Reproduce
1. Export configuration from another airbyte instance
2. Deploy airbyte to kubernetes (this step is probably not relevant)
3. In Kubernetes Airbyte, attempt to import the configuration

## Are you willing to submit a PR?
No

**Comment by jrhizor** [comment]
A few questions:
1. Does it work if you retry it?
2. Are there any error logs in the webapp Chrome Dev Tools console?
3. Can you share the server log surrounding the attempted import?

**Comment by spacecowboy** [comment]
1. Retried multiple times. Sometimes I drag n dropped, sometimes I browsed to the file. No difference
2. Couldn't see anything but I may not have looked in the right place
3. No I've already nuked it. The delay between actions because it needs to spin up new pods for every little thing made it completely unusable. I've since moved to a single dedicated GoogleCompute instance and that actually works.
